src/main/zapHomeFiles/hud/display.js

```diff
@@ -617,7 +617,7 @@ Vue.component('websocket-message-modal', {
 
 		eventBus.$on('showWebSocketMessageModal', data => {
 			const date = new Date(Number(data.msg.timestamp));
```

Comment:
I think the `date` object should also be extracted into the new function, it's only used for that.

Reply:
Verified and updated the relevant files.